docs/core_docs/docs/integrations/chat/groq.ipynb

```diff
@@ -103,7 +103,10 @@
     "    maxTokens: undefined,\n",
     "    maxRetries: 2,\n",
     "    // other params...\n",
-    "})"
+    "})\n",
+    "\n",
+    "// if you need a response format other than the default\n",
+    "const llmWithResponseFormat = llm.bind({ response_format: \"json_object\" });"
```

Comment:
Or wait, does this need to be `response_format: { type: "json_object" }`? I think that's what OpenAI does

Reply:
Yes
<img width="832" alt="image" src="https://github.com/user-attachments/assets/694dbdb1-6363-4984-a177-147e64ed5f9d">